server/src/rooms/game/events/index.ts

```diff
@@ -310,6 +310,7 @@ export class EnteredDefeatPhase extends GameEventWithData {
 
   apply(game: GameState): void {
     game.phase = Phase.defeat;
+    game.timeRemaining = 9999999;         // set timeRemaining = infinite to prevent phase transitioning after game is over
```

Comment:
I see why this is happening but should we instead exit from the gameLoop entirely and dispose the room when the game is over and everything has been finalized / properly persisted? @cpritcha I seem to recall you had some commits to fix GameRoom disposal..

Reply:
in the gameLoop we now call disconnect immediately once the game phase is defeat or victory.